Assets/CesiumForUnitySamples/Scripts/CesiumSamplesRequiresMagicLeap.cs

```diff
@@ -0,0 +1,243 @@
+#if UNITY_EDITOR
+using System.Linq;
+using UnityEditor;
+using UnityEditor.XR.Management;
+using UnityEditor.XR.Management.Metadata;
+using UnityEditor.XR.OpenXR.Features;
+#endif
+using UnityEngine;
+#if UNITY_EDITOR
+using UnityEngine.XR.Management;
+using UnityEngine.XR.OpenXR;
+using UnityEngine.XR.OpenXR.Features.MetaQuestSupport;
+using UnityEngine.XR.OpenXR.Features;
+using System;
+using static UnityEditor.XR.OpenXR.Features.OpenXRFeatureSetManager;
+using UnityEngine.XR;
+
+#if CESIUM_MAGIC_LEAP
+using UnityEngine.XR.OpenXR.Features.MagicLeapSupport;
+#endif
+#endif
+
+[ExecuteInEditMode]
+public class CesiumSamplesRequiresMagicLeap : MonoBehaviour
+{
+#if UNITY_EDITOR
+    private const string ML_PACKAGE_REQUIRED_TEXT =
+         "The Magic Leap SDK needs to be installed in the project for this sample to work. " +
+         "Click below to read instructions from the Magic Leap documentation on how to add the SDK to the project.";
+    private const string ML_PACKAGE_DOCS_LINK = "https://developer-docs.magicleap.cloud/docs/guides/unity/getting-started/configure-unity-settings/#import-magic-leap-unity-sdk";
+
+    private const string ML_BUILD_SETTINGS_TEXT =
+        "Build settings need to be changed to support the Magic Leap. Click Ok to perform these changes automatically.";
+    private const string ML_MIN_VERSION_TEXT =
+        "The Magic Leap sample requires Unity version 2022.2 or greater. Please reopen the project in a compatible version of Unity to continue.";
```

Comment:
I would strength this check to Unity 2022.3.11 or higher. Because in previous versions of Unity, the Magic Leap package had a camera bug that was preventing 3D Tiles from loading correctly. (the camera would report 1x1 dimensions, which is less than ideal for our screen space loading algorithm)